packages/terra-card/src/Card.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import './Card.scss';
+
```

Comment:
To create name-spaced class names for css modules, please update
`import './Card.scss';` to `import styles from './Card.scss';`
 and then add
`const cs = classNames.bind(styles);`

To incorporate this style binding, you might need to update how you are applying classNames for vertical and horizontal padding. [Here](https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx) is an example that would be good to look at for guidance.